### PR TITLE
Add Post TC election checklist

### LINKED ISSUE #146: Create post election TC checklist
### Description
We need a checklist describing what steps to perform after a new Technical Committee in put in place. It should contain the action to (re)elect Security Officers.

### Motivation
To not forget any important steps to be performed by the new TC

### Exemplification
N/A

### Benefits
Obvious

### Possible Drawbacks
None

### PR DESCRIPTION
**Pull request by m-linner-ericsson**
### Applicable Issues

Fixes #146 

### Description of the Change

Added a TC post election checklist in the `howto` folder containing tasks to perform

### Alternate Designs

We could have merged this with https://github.com/eiffel-community/community/blob/master/howtos/update-tc-members.md but that list covers needs for one person whereas this list covers task for the TC.

### Benefits

To not forget any important steps to be performed by the new TC

### Possible Drawbacks

None that I can think of

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Mattias Linnér, mattias.linner@ericsson.com

### CODE REVIEW

File: howtos/post-tc-election-checklist.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Post Technical Election Checklist
+
+After an election of the Technical Committee (TC) perform the following:
+
+- [ ] For new member of the TC perform the actions ine [Update TC members](./update-tc-members.md)
```
Comment (t-persson) [inline howtos/post-tc-election-checklist.md:5]:
```suggestion
- [ ] For new member of the TC perform the actions in [Update TC members](./update-tc-members.md)

```

File: howtos/post-tc-election-checklist.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] For leaving members of the TC undo the actions in [Update TC members](./update-tc-members.md)
+- [ ] Perform election of security officers
+  - [ ] Announce it on the community mailing list 
+  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers with the new officers
+- [ ] Perform election of elections officers
+  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#election-officers with the new officers
+  - [ ] Announce it on the community mailing list 
+- [ ] TC chairs to call for TC meetings
+- [ ] TC charis to call for the monthly community meetings
```
Comment (t-persson) [inline howtos/post-tc-election-checklist.md:14]:
```suggestion
- [ ] TC chairs to call for the monthly community meetings

```

File: howtos/post-tc-election-checklist.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Post Technical Election Checklist
```
Comment (e-backmark-ericsson) [inline howtos/post-tc-election-checklist.md:1]:
```suggestion
# Post Technical Committee Election Checklist

```

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]

File: howtos/post-tc-election-checklist.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+- [ ] For leaving members of the TC undo the actions in [Update TC members](./update-tc-members.md)
+- [ ] Perform election of security officers
+  - [ ] Announce it on the community mailing list 
+  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers with the new officers
+- [ ] Perform election of elections officers
+  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#election-officers with the new officers
+  - [ ] Announce it on the community mailing list 
+- [ ] TC chairs to call for TC meetings
+- [ ] TC chairs to call for the monthly community meetings
```
Comment (magnusbaeck) [inline howtos/post-tc-election-checklist.md:16]: "Call for" isn't the write verb here. In this context it reads as "TC chairs demand TC meetings".

```suggestion
- [ ] TC chairs send invite to TC meetings
- [ ] TC chairs send invite to the monthly community meetings
```

Also, should we specify the recipient(s) of the invites?

**Review by magnusbaeck — CHANGES_REQUESTED** [review]

File: howtos/post-tc-election-checklist.md (hunk 4 above)
Comment (m-linner-ericsson) [reply howtos/post-tc-election-checklist.md]: Fixed in [1e676eb](https://github.com/eiffel-community/community/pull/180/commits/1e676eb807b0664284321b3a3ec49e646c323ab9)

**Review by magnusbaeck — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

**Pull request merged**